web/src/components/molecules/Asset/Asset/AssetBody/UnzipFileList.tsx

```diff
@@ -65,18 +78,57 @@ const UnzipFileList: React.FC<Props> = ({ file, assetBaseUrl, setAssetUrl, style
   };
 
   return (
-    <Tree
-      switcherIcon={<Icon icon="caretDown" />}
-      expandedKeys={[...expandedKeys]}
-      selectedKeys={[...selectedKeys]}
-      onSelect={onSelect}
-      onExpand={onExpand}
-      treeData={treeData}
-      style={style}
-      multiple={false}
-      showLine
-    />
+    <UnzipFileListWrapper>
+      {archiveExtractionStatus === "in_progress" ? (
+        <Spin tip={t("Decompressing...")} size="large" />
+      ) : archiveExtractionStatus === "failed" ? (
+        <ExtractionFailedWrapper>
+          <ExtractionFailedIcon icon="closeCircle" color="#FF4D4F" size="56px" />
+          <ExtractionFailedText>
+            {t("Failed to decompress, please check the uploaded file.")}
+          </ExtractionFailedText>
+        </ExtractionFailedWrapper>
+      ) : (
+        <Tree
+          switcherIcon={<Icon icon="caretDown" />}
+          expandedKeys={[...expandedKeys]}
+          selectedKeys={[...selectedKeys]}
+          onSelect={onSelect}
+          onExpand={onExpand}
+          treeData={treeData}
+          multiple={false}
+          showLine
+        />
+      )}
+    </UnzipFileListWrapper>
   );
 };
 
+const UnzipFileListWrapper = styled.div`
+  height: 250px;
+  overflow-y: scroll;
+  background-color: #f5f5f5;
+`;
+
+const ExtractionFailedWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+`;
+
+const ExtractionFailedIcon = styled(Icon)`
+  margin-bottom: 28px;
+`;
+
+const ExtractionFailedText = styled.span`
```

Comment:
Can you please only use span when you want to style text inside another text element. Span shouldn't be used for the full text.
Text should be the usually h1, h2, ..., p.

Reply:
I used <p> instead, Done!

Reply:
Thanks! I typically use the p tag as well for most non-title text, so this is good. But just remember p tags come with margin, so if you don't want margin, make sure to do `margin: 0;` to keep text aligned nicely

Reply:
I added `margin-bottom: 0;`
Done!